### PR TITLE
SDRLNE-4

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: newsroom/default_settings.py [hunk 1]
```diff
@@ -108,13 +108,13 @@
     'newsroom.media_utils',
 ]
 
-SITE_NAME = 'AAP Newsroom'
-COPYRIGHT_HOLDER = 'AAP'
+SITE_NAME = 'Медиасеть'
```
Comment (petrjasek) [inline newsroom/default_settings.py]: you shouldn't change those default_settings, but rather add `settings.py` file to root folder and put those changes there, like in https://github.com/superdesk/newsroom/blob/ansa/settings.py

Comment (darconny) [reply newsroom/default_settings.py]: Thanks please check now.

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import os
```
Comment (petrjasek) [inline settings.py]: you can remove this line, travis fails because of it

**Comment by darconny** [comment]
@petrjasek Please check now and merge if its OK.

**Pull request merged**